HHS/Head-Start-TTADP
# [TTAHUB-215/101] Add accessibility option for TTA Hours graph/change app name to TTA Hub

## Description of change

Add a button that toggles the TTA Hours Widget between being a graph and a table.

Changed app name on the frontend and in email templates

## How to test
Visit the regional dashboard page. Click on the link that says "view accessible data" next to the title and time on the TTA Hours widget. Note that the data changes and is correct.

Confirm that both the login modal and the upper left text in the header have the correct text.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-215
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-110

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
